Frontend/ui-customer/src/Helpers/Utils.purs

```diff
@@ -729,4 +730,18 @@ getExistingTags savedLoc = map (\item -> DS.toLower $ item.tag) savedLoc
 -- , recencyDate : Nothing
 -- , locationScore : Nothing
 
--- }) savedLocation )
\ No newline at end of file
+-- }) savedLocation )
+
+getCityConfig :: Array CityConfig -> String -> CityConfig
```

Comment:
these config functions should be there in config related files instead of utils?